spec/translator_spec.lua

```diff
@@ -0,0 +1,341 @@
+local util = require "pallene.util"
+
+local function compile(pallene_code)
+    assert(util.set_file_contents("__test__.pln", pallene_code))
+    local ok, _, _, error_message = util.outputs_of_execute("./pallenec __test__.pln --emit-lua")
+    if not ok then
+        error(error_message)
+    end
+end
+
+local function assert_translation(pallene_code, expected)
+    compile(pallene_code)
+    local contents = util.get_file_contents("__test__.lua")
+    assert.are.same(contents, expected)
```

Comment:
IIRC, the expected "should" come before the "contents". This makes a difference in the error message that the asser generates. Please check to see if they are the right way around.